src/compiler/binder.ts

```diff
@@ -455,7 +456,8 @@ function getModuleInstanceStateForAliasTarget(specifier: ExportSpecifier, visite
     return ModuleInstanceState.Instantiated; // Couldn't locate, assume could refer to a value
 }
 
-const enum ContainerFlags {
+/** @internal */
+export const enum ContainerFlags {

```

Comment:
I'm not experienced enough to know the right way to do this, but it does feel a little suspicious to export... I feel like I've seen a function that does something similar in the checker but I can't recall the name.

Reply:
I wanted to make sure the `container` used to check the implicit export rules is precisely the same the binder uses, and the binder uses those flags for that... I saw a `getContainerNode` in services/utilities, and `getEnclosingBlockScopeContainer` in compiler/utilities, but those are different.